projects/project-player-library/src/lib/services/utils/utils.service.ts

```diff
@@ -39,4 +39,19 @@ export class UtilsService {
     let response = await firstValueFrom(dialogRef.afterClosed())
     return response
   }
+
+  newWindow(dataResponse:any){
```

Comment:
Change the function name from newWindow to previewFile or viewFile something like that

Reply:
done